app/models/broken_window/measured_metric.rb

```diff
@@ -20,6 +20,17 @@ def status
       @status ||= calculate_status
     end
 
+    def unique_daily_measurements
```

Comment:
why is this unique? if value is the same for 2 days, then it would not include it?